src/info/LiquidatorInfo.sol

```diff
@@ -127,16 +127,29 @@ contract LiquidatorInfo is Math {
     function getCushionInfo(uint cdp, address me, uint numMembers) public view returns(CushionInfo memory info) {
         CdpDataVars memory c;
         (c.cdpArt, c.cushion, c.cdpWinners, c.bite) = pool.getCdpData(cdp);
-        info.isToppedUp = c.cushion > 0;
+        info.isToppedUp = manager.cushion(cdp) > 0;
         bool isUntoppedByUser = manager.cushion(cdp) == 0;
 
+        bool mustCallUntop = c.cushion > 0 && isUntoppedByUser;
+
+        if(info.isToppedUp && isUntoppedByUser || mustCallUntop) {
+            for(uint i = 0 ; i < c.cdpWinners.length ; i++) {
+                if(me == c.cdpWinners[i]) {
+                    uint perUserArt = c.cdpArt / c.cdpWinners.length;
+                    if(perUserArt > c.bite[i] || mustCallUntop) {
```

Comment:
here `mustCallUntop` is always true. but the `if` should now always be executed.
IMO the only condition here should be `perUserArt > c.bite[i]`.